test/cmd/dnshammer.go

```diff
@@ -94,11 +102,34 @@ func do(addr string, n int, records map[string][]string) error {
 	return nil
 }
 
-func ipToString(ips []net.IP) (out []string) {
-	for _, ip := range ips {
-		out = append(out, ip.String())
+func doTXT(addr string, n int, records map[string][]string) error {
```

Comment:
This is almost the same as `doA`; can we merge the two (and passing in a lookup function returning `[]string, error`, plus a compare function of type `func([]string, []string) bool`)?

Reply:
I considered merging these, however, the lookup functions have a different argument signature and in addition, we would need to add more lookup funcs in the close future, e.g lookupMX, lookupSRV, lookupHost, etc. They will all have a diffrent signature, if we refactor now, we would need to decouple this shortly.